conf/celeborn-ratis-shell-env.sh.template

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
we should grant `x` permission to this all `*.sh` or `*.sh.template` file

Reply:
Removed this file, can ignore this now